reconcile/gitlab_permissions.py

```diff
@@ -50,6 +52,19 @@ def run(dry_run, thread_pool_size=10, defer=None):
     if defer:
         defer(gl.cleanup)
     repos = queries.get_repos(server=gl.server, exclude_manage_permissions=True)
+    toggle_state = get_feature_toggle_state(
+        "gitlab-permissions-share-with-group",
+        default=False,
+    )
+    if not toggle_state:
+        share_project_with_group(gl, repos, dry_run)
+    else:
+        share_project_with_group_members(gl, repos, thread_pool_size, dry_run)
```

Comment:
When `toggle_state` is `True`, it should mean `share_project_with_group`, but `if` condition is in the opposite, better to rename it to something meaningful like `should_share_project_with_group` or just inline condition check

Reply:
@mehfuz what I mean is variable name is not meaningful, by looking at `toggle_state`, can't tell what does it represent, feature toggle name is ok